### PR TITLE
Vam 520 orwdal32 load for edit

### PR DESCRIPTION
**Pull request by kekuso**
- punted several pieces due to lack of data
- dates seem mis-implemented in the MUMPS code

### CODE REVIEW

File: emulation/models/orwdal32-load-for-edit.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    const inputArr = input.split(splitter);
+    const acronym = inputArr.map(word => word[0]);
+    return lowerCase ? acronym.join('').toLowerCase() : acronym.join('');
+};
+
+const RPC_EMULATOR_MODEL = [
+    {
+        id: 'ORWDAL32_LOAD_FOR_EDIT',
+        name: 'ORWDAL32 LOAD FOR EDIT',
+        function: 'EDITLOAD^ORWDAL33',
```
Comment (mfuroyama) [inline emulation/models/orwdal32-load-for-edit.js]: Really minor: this should be `'EDITLOAD^ORWDAL32'`

File: emulation/models/orwdal32-load-for-edit.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        parameters: [],
+        files: ['120.8'],
+        domain: 'ALLERGY',
+        labels: ['PATIENT', 'ALLERGY'],
```
Comment (mfuroyama) [inline emulation/models/orwdal32-load-for-edit.js]: You may want to pre-emptively add a `'RESULTS-UNCHECKED'` here

File: emulation/models/orwdal32-load-for-edit.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        returnType: 'ARRAY',
+        inputParameters: [{
+            id: 'allergyId',
+            parameterType: 'LITERAL',
+            required: true,
+            description: 'The allergy ID to lookup',
+            example: '1',
+        }],
+        invokeRPC: (args, session, callback) => {
+            const allergyId = args[0] || '';
```
Comment (mfuroyama) [inline emulation/models/orwdal32-load-for-edit.js]: Don't forget to convert this to a URN

File: emulation/models/orwdal32-load-for-edit.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+                '~OBS/HIST',
+                `d${createAcronym(res.observedHistorical, ' ', true)}^${res.observedHistorical.toUpperCase()}`,
+                '~OBSERVATIONS', // punt - more stuff pushed into finalRes if prev value is HISTORICAL
+                'tRecord            : 1', // punt - the rest of this is related to File 120.85 ADVERSE REACTION TRACKING
+                'tDate/Time of Event: FEB 21,2018',
+                'tObserver          : ',
+                'tSeverity          : MODERATE',
+                'tDate Reported     : ',
+                'tReporting User    : ',
+                't------------------------------------------------------------',
```
Comment (mfuroyama) [inline emulation/models/orwdal32-load-for-edit.js]: I know, this is gonna sound stupid, but do me a solid, and replace the word `punt` in this section with `to investigate`

File: emulation/models/orwdal32-load-for-edit.js (hunk 1 above)
Comment (kekuso) [reply emulation/models/orwdal32-load-for-edit.js]: ahhh good catch lol

File: emulation/models/orwdal32-load-for-edit.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+                '~VERIFIER', // punt
+                '~VERIFIED', // punt
+                'dYES^3180221.02543', // punt (Verification date?)
+                '~ENTERED IN ERROR',
+                `d${res.enteredInError ? 'YES' : 'NO'}`, // half-punt, need Entered in Error data to test
+                '~OBS/HIST',
+                `d${createAcronym(res.observedHistorical, ' ', true)}^${res.observedHistorical.toUpperCase()}`,
+                '~OBSERVATIONS', // punt - more stuff pushed into finalRes if prev value is HISTORICAL
+                'tRecord            : 1', // punt - the rest of this is related to File 120.85 ADVERSE REACTION TRACKING
+                'tDate/Time of Event: FEB 21,2018',
```
Comment (mfuroyama) [inline emulation/models/orwdal32-load-for-edit.js]: These date fields might be too far to punt. :) We might have to at least add some credible empty data structures to this thing.

**Pull request merged**